go.sum

```diff
@@ -1,13 +1,2 @@
-github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
thanks, the changes to this file are a good illustration of the transitive deps k8s.io/utils would have imposed on all consumers of klog

inlining (especially if this was only used in a test) sgtm